client/i18n/locales/english/intro.json

```
@@ -301,68 +301,113 @@
   },
   "2022/javascript-algorithms-and-data-structures": {
     "title": "JavaScript Algorithms and Data Structures (Beta)",
-    "intro": ["placeholder", "placeholder"],
-    "note": "",
+    "intro": [
+      "While HTML and CSS control the content and styling of a page, JavaScript is used to make it interactive. In the JavaScript Algorithm and Data Structures Certification, you'll learn the fundamentals of JavaScript like variables, arrays, objects, loops, functions, the DOM and more.",
+      "In addition to the fundamentals, you will learn about Object Oriented Programming (OOP), Functional Programming (FP), algorithmic thinking, how to work with local storage and how to fetch data from an API."
+    ],
+    "note": "Note: Some browser extensions, such as ad-blockers and script-blockers can interfere with the tests. If you face issues, we recommend disabling extensions that modify or block the content of pages while taking the course.",
     "blocks": {
       "build-a-caesars-cipher-project": {
         "title": "Build a Caesars Cipher Project",
         "intro": ["", ""]
       },
       "build-a-cash-register-project": {
         "title": "Build a Cash Register Project",
-        "intro": ["", ""]
+        "intro": [
+          "In this certification project, you will need to build a functioning cash register.  Your project should take in an input of the user cash and output the status of the cash register as well as any change due, if necessary.",
+          "You will need to use HTML, CSS and JavaScript and pass all of the required user stories."
```

```suggestion
          "This is one of the required projects to earn your certification.",
          "For this project, you will build a cash register web app."
```

I think we should follow the same format we did for the RWD cert project intros.